pkgs/by-name/ul/ultraleap-hand-tracking-control-panel/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, dpkg
+, makeWrapper
+, breakpointHook
+, autoPatchelfHook
+, libgcc
+, zlib
```

```suggestion
, autoPatchelfHook
, breakpointHook
, dpkg
, libgcc
, makeWrapper
, zlib
```
It's best practice for dependencies to be alphabetized.